apps/test/unit/code-studio/components/progress/teacherPanel/StudentTableTest.jsx

```diff
@@ -109,7 +109,7 @@ describe('StudentTable', () => {
       students: [
         {id: 1, name: 'Student 1', familyName: 'FamNameB'},
         {id: 2, name: 'Student 2'},
-        {id: 3, name: 'Student 3', familyName: 'FamNameA'},
+        {id: 3, name: 'Student 3', familyName: 'ZFamNameA'},
```

Comment:
Just to make sure because I'm not as familiar with this keyword - 'match' here on the lines below is more like a 'contains' than an 'equals', yes?

Reply:
Yep! The full text of these rows is something like "Student 3 ZFamNameA<FontAwesome />". It seems the prior test authors didn't want to type all that for a full equality check, and I'm happy to follow their lead on that.

Reply:
no no this works! Just making sure I'm following:)